frontend/packages/dev-console/integration-tests/utilities/appFunctions.ts

```diff
@@ -0,0 +1,71 @@
+import {click} from './elementInteractions';
+import {$, browser, ExpectedConditions as EC} from 'protractor';
+const waitForElement = 5000;
+
+export enum NavigationMenu {
+    Add = '+Add',
+    Topology = ' Topology',
+    Monitoring = 'Monitoring',
+    Builds = 'Builds',
+    Search = 'Search',
+    Helm = 'Helm',
+    ProjectDetails = 'Project Details',
+    ProjectAccess = 'Project Access'
+  }
+
+export const naviagteTo = async function(opt: NavigationMenu) {
+    switch (opt) {
+      case NavigationMenu.Add: {
+        await click($('[data-test-id="+Add-header"]'));
```

Comment:
can we use this function `elementByDataTestID` where ever we use [data-test-id="test-id] ?

Reply:
Sure, I will use this function. from next PR onwards.